client/utils/output.py

```diff
@@ -96,3 +96,19 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         if self.re_enable:
             on()
+
+class DisableStdout:
+    stdout_ref = None
+    null_file = None
+    def __init__(self):
+        if DisableStdout.stdout_ref is None:
+            DisableStdout.stdout_ref = sys.stdout
+
+    def __enter__(self):
+        f = open(os.devnull, 'w')
+        sys.stdout = f
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        # sys.stdout.close()
+        sys.stdout = DisableStdout.stdout_ref
```

Comment:
what's the use case for this? i don't see it used in this PR, except for this definition

Reply:
Used in our faded parsons repo to silence doctest output. Do you think we should move it there?

Reply:
yes please -- since it's only used by fpp, it should go in the fpp repo

Reply:
ok sounds good